protocol-units/execution/dof/src/v1.rs

```diff
@@ -50,7 +51,17 @@ impl DynOptFinExecutor for Executor {
 	async fn run_background_tasks(&self) -> Result<(), anyhow::Error> {
 		loop {
 			// readers should be able to run concurrently
-			self.executor.tick_transaction_pipe(self.transaction_channel.clone()).await?;
+			match self.executor.tick_transaction_pipe(self.transaction_channel.clone()).await {
+				Ok(_) => {}
+				Err(e) => match e {
+					TransactionPipeError::TransactionNotAccepted(e) => {
+						// allow the transaction not to be accepted by the mempool
+						// because the client may have sent a bad sequence number
+						tracing::error!("Transaction not accepted: {:?}", e);
```

Comment:
The error severity level is meant for fatal things. If this is to be handled in the normal course of operation, but logged as per default log settings, this should be `info!` or maybe `warn!`.

Reply:
Sorry, yes, I'll change to `warn!`.